demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -1035,6 +1039,15 @@ def are_valid_files(self, id_set_validations) -> str:
 
     # pack dependencies validation
     def validate_pack_dependencies(self):
+
+        if self.validate_graph:
+            click.secho(
+                f"\nRunning pack dependencies validation on {self.pack} with the graph\n",
+                fg="bright_cyan",
+            )
+            with GraphValidator(self.specific_validations) as graph_validator:
+                graph_validator.validate_dependencies(self.pack)
+
```

Comment:
```suggestion
            with GraphValidator(self.specific_validations) as graph_validator:
                graph_validator.validate_dependencies(self.pack)
        else:
```
if we do graph validation, don't do id set validation.

Reply:
also I think better to do all graph validations in one place (under is_valid_content_graph), and here do the ID set validation only if `not self.validate_graph`